### ISSUE TITLE
Some suggestions

### ISSUE THREAD

**Issue by ghost** [lian/msfs2020-go#21]
It would be nice if you could fold that upper area with flight instruments(alt, speed and etc). Also would be nice if you can access different flight instruments with url like  localhost:9000/Alt for altimeter  , localhost:9000/Speed for speed and etc(to put where you want), and maybe with graphical visualization like https://github.com/sebmatton/jQuery-Flight-Indicators or https://github.com/uw-ray/Skyhawk-Flight-Instruments or https://codesandbox.io/s/six-pack-flight-instruments-fl7b8?from-embed

**Comment by 1a57danc3** [comment]
https://www.figma.com/file/PvBsaPhBlwX5dR9Q8vQ59J/msfs2020-go?node-id=5%3A475

Web UI/UX improvement plan, Do you like it?

**Comment by Rukenshia** [comment]
@h0wardch3ng are you going to implement that yourself? otherwise I'm happy to help out on that.

In the mean time, this is the current index.html I'm running https://gist.github.com/Rukenshia/f225191a0a215a856f1f6b68850c8527. It already allows toggling between "basic" and "full" aircraft information, but not hiding it quite yet as I just noticed this issue with the nice design 😄 

basic:
![image](https://user-images.githubusercontent.com/1899599/91652319-693fbc00-ea96-11ea-83b2-735423647ad4.png)


full:
![image](https://user-images.githubusercontent.com/1899599/91652309-5200ce80-ea96-11ea-90e0-c9c63dfb923b.png)

**Comment by 1a57danc3** [comment]
@Rukenshia I'm not a professional front-end developer, but I can offer UI/UX assistance, and I think this project is a great idea, and I can make this flight simulator tool simple and beautiful!

Features I currently feel need to be added

1. Screen horizontal and vertical (iPad) adaptations
2. Night Mode(mapbox offer theme maker function, that's easy to make)
3. Online map provider (openstreetmap or mapbox) switch
4. Local Settings Storage (User settings recorded in local Cookie or Web Storage)

Future prospects for this tool
1. Add the appearance of Windows launcher, not the current CLI program.

If you're interested, contact me.
https://t.me/h0wardch3ng

**Comment by Rukenshia** [comment]
I'm just another random internet person trying to help out, so if you're cool with it @lian I'd get started on getting that design implemented. One thing I was wondering where exactly you got the `_vendor` assets from as they're not checked into the code.

I'm not sure yet about the integration of Mapbox as I haven't done that before, but I'll try that once I've got the updated UI going 👍

**Comment by lian** [comment]
@Rukenshia @h0wardch3ng totally cool with it. i want others to contribute to this project aswell! haven't done much front-end code in years and super happy if someone else can do it.

i like @xoxaxo's idea of re-using those flight instrument js libs to render them. but i also like the simple values display. somehow it should be configurable so people either gets instruments, raw text, or everything hidden. maybe even the position where what is shown, and what values are shown. and that should be saved in your web session (cookie) or in a server side config file.

yesterday received another suggestion that we should also show ground speed besides airspeed

i wanted to look at getting the data of other planes (multiplayer) and waypoints next.

**Comment by 1a57danc3** [comment]
@lian 
You can get waypoints data from this site (login required)
https://www.simbrief.com/system/dbquery.php?target=map

**Comment by F7DRJ** [comment]
@lian Loving the app! Have a few questions though. Any way that I could use Google maps instead of open street view? Like the look a lot better. Secondly do you plan on implementing navigational data such as VORs/Coms data?

**Comment by lian** [comment]
@F7DRJ google maps, sadly no. discussed here https://github.com/lian/msfs2020-go/issues/9
for vor/coms data did you try to select navigational data option in the upper right corner of the map?

**Comment by F7DRJ** [comment]
> @F7DRJ google maps, sadly no. discussed here #9
> for vor/coms data did you try to select navigational data option in the upper right corner of the map?

Understandable. An yes I've toggled navigational data but i dont see VOR frequencies. Maybe I'm missing something.

**Comment by lian** [comment]
@F7DRJ yes, it only uses (and shows) what http://maps.openaip.net/ provides.

**Comment by F7DRJ** [comment]
@lian Gotcha. And thanks again for this awesome software. Looking forward to updates on it, or anything else you may bring to the table.

**Comment by w4mk** [comment]
I am new at sims but I got a flight loaded and got your map on my chrome browser to work but how do I put it on the same screen where I am flying the plane in the sim?  Like the N button map comes up??

**Comment by ip999** [comment]
you could try using a chrome extension such as [this](https://chrome.google.com/webstore/detail/flobro-floating-browser-w/faaljkdndnfoagcmhedlmbgieoocemch?hl=en), not sure how much impact it will have on framerate though.

**Comment by andrewlilley-ux** [comment]
Hi, I think I’m your man for the UI and UX ;) I’m an experienced UX designer, and have worked for two aerospace giants, helping design cockpit displays and tactical mission systems. This little project is just my cup of tea :)

I don’t do coding, so I would contribute by providing user needs, interface needs, designs and interactive prototypes, and maybe testing if we can find any willing users. Would this work for you?

I think this app has massive potential.

**Comment by Rukenshia** [comment]
Started with the implementation based on @h0wardch3ng design here: https://github.com/Rukenshia/msfs2020-go/tree/feature/redesign

Here's some WIP images:

[light mode, big screen](https://user-images.githubusercontent.com/1899599/92077597-082d2680-edbd-11ea-9cad-a94a5874e51a.png)
[dark mode, big screen, with settings open](https://user-images.githubusercontent.com/1899599/92077619-124f2500-edbd-11ea-85e4-4d94a92d2eee.png)

On smaller screens, it looks like this:

[dark mode, small screen](https://user-images.githubusercontent.com/1899599/92077657-27c44f00-edbd-11ea-87c4-3f31e40b3b2b.png)

**Comment by lian** [comment]
@Rukenshia awesome, :panda_face: looking forward to pull in those changes!

@andrewlilley-ux thank you for your comment. would this mean working html, or maybe we find someone willing to convert it into working frontend html then? i'm personally looking into adding some backend features like other aircraft traffic (might not be possible with the current simconnect :disappointed:) and still waypoints, and  update note if a new version is released and such. its great that others have picked up the frontend part to make the UX way nicer.

we also need to make the follow/don't-follow plane nicer. on twitch and youtube users have a hard time and think the map is always fixed to the plane. maybe a startup box that shows if an update is available, how to use the app, credits to contributors.

**Comment by andrewlilley-ux** [comment]
@lian Maybe we can do with using a Kanban board? Trello, for example? We could have columns for Design Icebox, Design Backlog, Design Doing, Design Ready, Dev Icebox, Dev Backlog, Dev Doing, Dev Done.

**Comment by andrewlilley-ux** [comment]
I'm going to work on design suggestions for the functions listed by @h0wardch3ng, @Rukenshia and @lian, and make a clickable prototype. @h0wardch3ng hope you're OK with that - we can bounce ideas of each other.

**Comment by andrewlilley-ux** [comment]
and your suggestions of course, @xoxaxo.

**Comment by F7DRJ** [comment]
Any way to view this on my phone? That would be dope! It's an Android if that matters...
